cli/lib/kontena/cli/stacks/yaml/custom_validators/hooks_validator.rb

```diff
@@ -6,10 +6,16 @@ def initialize
 
     def validate(key, value, validations, errors)
       unless value.is_a?(Hash)
-        errors[key] = 'hooks must be array'
+        errors[key] = 'hooks must be hash'
```

Comment:
also most of the errors seem to say something like `must be string` , should be `must be a string`.